### PR TITLE
issue #20 - changing visibility of log messages

### PR DESCRIPTION
**Pull request by filipc**

### CODE REVIEW

File: library/src/main/java/com/pixplicity/sharp/Sharp.java [hunk 1]
```diff
@@ -1724,7 +1724,7 @@ private void finishGradients() {
                     positions[i] = gradient.mPositions.get(i);
                 }
                 if (colors.length == 0) {
-                    Log.w(TAG, "bad gradient, id=" + gradient.mId);
+                    Log.d(TAG, "bad gradient, id=" + gradient.mId);
```
Comment (pflammertsma) [inline library/src/main/java/com/pixplicity/sharp/Sharp.java]: I disagree that this should be a debug message. The gradient is invalid, and a warning should be logged.

File: library/src/main/java/com/pixplicity/sharp/Sharp.java [hunk 2]
```diff
@@ -1923,7 +1923,7 @@ public void startElement(String namespaceURI, String localName, String qName, At
                 if (width < 0 || height < 0) {
                     width = 100;
                     height = 100;
-                    Log.w(TAG, "element '" + localName + "' does not provide its dimensions; using " + width + "x" + height);
+                    Log.d(TAG, "element '" + localName + "' does not provide its dimensions; using " + width + "x" + height);
```
Comment (pflammertsma) [inline library/src/main/java/com/pixplicity/sharp/Sharp.java]: I have the same objection here.

**Comment by filipc** [comment]
Now it should fit

**Comment by pflammertsma** [comment]
Thanks for your contributions!

**Pull request merged**